treeherder/etl/jobs.py

```diff
@@ -335,13 +336,22 @@ def _schedule_log_parsing(job, job_logs, result):
     # RabbitMQ priority feature (since the idea behind separate queues was
     # only to ensure failures are dealt with first if there is a backlog).
     if result != 'success':
-        queue = 'log_parser_fail'
-        priority = 'failures'
+        if job_lab.name == "errorsummary_json":
+            queue = "log_parser_fail_json"
+            priority = "failures"
+        else:
+            queue = "log_parser_fail_raw"
+            priority = "failures"
+        if repository.name in sheriffed_repos:
+            queue += "_sheriffed"
+        else:
+            queue += "_unsheriffed"
     else:
         queue = 'log_parser'
         priority = "normal"
 
-    parse_logs.apply_async(queue=queue, args=[job.id, job_log_ids, priority])
+    # parse_logs.apply_async(queue=queue, args=[job.id, job_log_ids, priority])
+    parse_logs(queue, job.id, job.log_ids, priority)
```

Comment:
`job.log_ids` should still be `job_log_ids`

Reply:
Thanks! fixed.